src/runtime_src/core/edge/tools/xbutil/CMakeLists.txt

```diff
@@ -10,15 +10,31 @@ file(GLOB XRT_EDGE_TOOLS_XBUTIL_FILES
 
 add_executable(xbutil ${XRT_EDGE_TOOLS_XBUTIL_FILES})
 
-target_link_libraries(xbutil
-  xrt_core_static
-  xrt_coreutil_static
-  ${Boost_FILESYSTEM_LIBRARY}
-  ${Boost_SYSTEM_LIBRARY}
-  pthread
-  rt
-  uuid
-  ${CURSES_LIBRARIES}
-  )
+if (DEFINED XRT_AIE_BUILD)
```

Comment:
Would it be better to simply leave the existing target_link_libraries alone and add a new AIE library.  For example:
```
if (DEFINED XRT_AIE_BUILD)
  target_link_libraries(xbutil xaiengine)
endif()
```